Dockerfile

```
@@ -1,60 +1,62 @@
-# Using a specific version for the base image
-FROM ubuntu:jammy
+FROM lsiobase/ubuntu:jammy as base
 
-# Creating a new user and group
-RUN adduser --system --ingroup users --disabled-login --gecos "diffusion user" --shell /bin/bash --uid 99 diffusion
+COPY docker/root/ /
 
-# Setting environment variables
 ENV DEBIAN_FRONTEND=noninteractive
 ENV WEBUI_VERSION=01
-ENV BASE_DIR="/opt/stable-diffusion"
-ENV SD_INSTALL_DIR="/opt/sd-install"
-ENV SD01_DIR="${BASE_DIR}/01-easy-diffusion"
-ENV SD02_DIR="${BASE_DIR}/02-sd-webui"
-ENV SD03_DIR="${BASE_DIR}/03-invokeai"
-ENV SD04_DIR="${BASE_DIR}/04-SD-Next"
-ENV SD05_DIR="${BASE_DIR}/05-comfy-ui"
-ENV SD06_DIR="${BASE_DIR}/06-Fooocus"
-ENV SD07_DIR="${BASE_DIR}/07-StableSwarm"
-ENV SD08_DIR="${BASE_DIR}/08-voltaML"
-ENV SD20_DIR="${BASE_DIR}/20-kubin"
-ENV SD50_DIR="${BASE_DIR}/50-lama-cleaner"
-ENV SD51_DIR="${BASE_DIR}/51-facefusion"
-ENV SD70_DIR="${BASE_DIR}/70-kohya"
-ENV XDG_CACHE_HOME="${BASE_DIR}/temp"
-
-# Installing required packages
+ENV BASE_DIR=/config
+ENV SD_INSTALL_DIR=/opt/sd-install
+ENV SD01_DIR=/config/01-easy-diffusion
+ENV SD02_DIR=/config/02-sd-webui
+ENV SD03_DIR=/config/03-invokeai
+ENV SD04_DIR=/config/04-SD-Next
+ENV SD05_DIR=/config/05-comfy-ui
+ENV SD06_DIR=/config/06-Fooocus
+ENV SD07_DIR=/config/07-StableSwarm
+ENV SD08_DIR=/config/08-voltaML
+ENV SD20_DIR=/config/20-kubin
+ENV SD50_DIR=/config/50-lama-cleaner
+ENV SD51_DIR=/config/51-facefusion
+ENV SD70_DIR=/config/70-kohya
+ENV XDG_CACHE_HOME=/config/temp
```

```suggestion
ENV BASE_DIR=/config \
    SD_INSTALL_DIR=/opt/sd-install \
    SD01_DIR=/config/01-easy-diffusion \
    SD02_DIR=/config/02-sd-webui \
    SD03_DIR=/config/03-invokeai \
    SD04_DIR=/config/04-SD-Next \
    SD05_DIR=/config/05-comfy-ui \
    SD06_DIR=/config/06-Fooocus \
    SD07_DIR=/config/07-StableSwarm \
    SD08_DIR=/config/08-voltaML \
    SD20_DIR=/config/20-kubin \
    SD50_DIR=/config/50-lama-cleaner \
    SD51_DIR=/config/51-facefusion \
    SD70_DIR=/config/70-kohya \
    XDG_CACHE_HOME=/config/temp
```
Not sure how meaningful it is, but this should result in a lot less layers to cache.